src/api/app/controllers/source_project_controller.rb

```diff
@@ -54,13 +55,11 @@ def render_project_packages
   def delete
     project = Project.get_by_name(params[:project])
 
+    raise Pundit::NotAuthorizedError unless project
```

Comment:
`.get_by_name` will already throw other exceptions in this case no?

Reply:
I'm not sure, I will add a test case for that.

Reply:
I will add a test case for that